actions/build-lambda-zip/action.yaml

```diff
@@ -31,10 +34,11 @@ runs:
       run: |
         echo "build zip..."
         make -C ${{ inputs.make-directory }} build
+    # TODO: Will delete this step in next PR.
     - name: Push to lambda registry s3
       id: push-zip
       shell: bash
       run: |
         echo "push zip..."
-        aws s3 cp ${{ inputs.repository }}.zip ${{ inputs.registry }}/${{ inputs.env }}/${{ inputs.repository }}.zip
+        # aws s3 cp ${{ inputs.repository }}.zip ${{ inputs.registry }}/${{ inputs.env }}/${{ inputs.repository }}.zip
```

Comment:
why remove s3 cp. this action is push Lambda zip

Reply:
We are not using this step. `sls` would take care of building the zip and pushing that to s3.